packages/@jsii/go-runtime/jsii-calc-test/main_test.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	calc "github.com/aws-cdk/jsii/jsii-calc/go/jsiicalc"
+	"github.com/aws-cdk/jsii/jsii-experimental"
+	"math"
+	"os"
+	"testing"
+)
+
+func TestMain(m *testing.M) {
+	defer jsii.Close()
+	os.Exit(m.Run())
```

Comment:
Does the `defer` statement actually run after `os.Exit()` has returned?

Reply:
Good catch, switched this around to correctly close.